### DISCUSSION TITLE
Statistics on a new tab on the web page

### DISCUSSION THREAD

**Discussion by riramar** [OWASP/www-project-secure-headers#210] [Ideas]
Hi @righettod 

You did an amazing job here https://github.com/oshp/oshp-stats/blob/main/stats.md.
What you think about creating a new Statistics tab here https://owasp.org/www-project-secure-headers/ and publish the same content as https://github.com/oshp/oshp-stats/blob/main/stats.md? Not sure if it's possible. It's just an idea.

Best regards,
Ricardo Iramar

**Comment by righettod** [comment]
Hi @riramar 

I will add it to the roadmap of 2025 😃

**Comment by righettod** [comment]
💻I started working on this point via the PR #217

**Comment by righettod** [comment]
💻Further work via PR #218

**Comment by righettod** [comment]
✅Done via the following PR:

- #217
- #218
- #219
- #220
- #221
